airflow/contrib/operators/azure_livysparksubmit_operator.py

```diff
@@ -0,0 +1,140 @@
+    :param main_class: main class to be submitted job
+    :type main_class: str
+    :param files:	files to be used in this session
+    :type files:	list[Str]
+    :param jars:	jars to be used in this session
+    :type jars:	list[str]
+    :param pyFiles:	Python files to be used in this session
+    :type pyFiles:	list[str]
+    :param driverMemory:	Amount of memory to use for the driver process
+    :type driverMemory:	str
+    :param driverCores:	Number of cores to use for the driver process
+    :type driverCores:	int
+    :param executorMemory:	Amount of memory to use per executor process
+    :type executorMemory:	str
+    :param executorCores:	Number of cores to use for each executor
+    :type executorCores:	int
+    :param numExecutors:	Number of executors to launch for this session
+    :type numExecutors:	int
+    :param archives:	Archives to be used in this session
+    :type archives:	str
+    :param queue:	The name of the YARN queue to which submitted
+    :type queue:	str
+    :param name:	The name of this session
+    :type name:	str
+    :param conf:	Spark configuration properties
+    :type conf:	dict
+    """
+
+    @apply_defaults
+    def __init__(self,
+                 ambari_conn_id='hortonworks_ambari_default',
+                 application_file=None,
+                 main_class=None,
+                 master=None,
+                 name=None,
+                 jars=None,
+                 py_files=None,
+                 files=None,
+                 executor_memory=None,
+                 executor_cores=None,
+                 num_executors=None,
+                 driver_memory=None,
+                 driver_cores=None,
+                 archives=None,
+                 queue=None,
+                 application_args=None,
+                 conf=None,
+                 execution_timeout: int = 5,
+                 *args,
+                 **kwargs
+                 ):
+
+        super(AzureLivySparkSubmitOperator, self).__init__(*args, **kwargs)
+
+        if conf is None:
+            conf = {}
+        if application_args is None:
+            application_args = []
+        if archives is None:
+            archives = []
+        if files is None:
+            files = []
+        if py_files is None:
+            py_files = []
+        if jars is None:
+            jars = []
+        self.ambari_conn_id = ambari_conn_id
+        self.files = files
+        self.pyFiles = py_files
+        self.jars = jars
+        self.name = name
+        self.queue = queue
+        self.archives = archives
+        self.numExecutors = num_executors
+        self.driverCores = driver_cores
+        self.executorCores = executor_cores
+        self.driverMemory = driver_memory
+        self.application_file = application_file
+        self.main_class = main_class
+        self.master = master
+        self.executorMemory = executor_memory
+        self.conf = conf
+        self.application_args = application_args
```

Comment:
This attribute is never used. Shouldn't it be passed to datas dictionary to be used as Spark job arguments?